eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -126,57 +157,67 @@ namespace eos
     }
 
     complex<double>
-    KKvDZ2022FormFactors::_calc_ff_contribution_omega(const double & q2, const double & k2, const double & m_r_sq, const std::array<UsedParameter, 3> & a) const
+    KKvDZ2022FormFactors::_calc_ff_contribution_omega(const double & q2, const double & k2, const double & m_r_sq, const std::array<UsedParameter, 3> & a,
+                                                                  const std::array<UsedParameter, 3> & c, const UsedParameter & s_0) const
     {
         double a_0(a[0]),a_1(a[1]),a_2(a[2]);
+        double c_0(c[0]),c_1(c[1]),c_2(c[2]);
         const double diff_z = _z_omega(k2) - _z_omega(0.0);
         const double m_omega = BToOmega::m_V, m_omega_sq = m_omega * m_omega;
 
-        return 1.0 / (1.0 - k2 / m_r_sq) * ( m_omega_sq / ( m_omega_sq - q2 - 1.0i * m_omega * _width_omega(q2) ) ) *
-                (a_0 + a_1 * diff_z + a_2 * power_of<2>(diff_z) );
+        return 1.0 / (1.0 - k2 / m_r_sq) * m_omega_sq / ( m_omega_sq - q2 - 1.0i * m_omega * _width_omega(q2) ) * ( (1.0 - switch_subtracted) *
+                (a_0 + a_1 * diff_z + a_2 * power_of<2>(diff_z) ) + switch_subtracted * (c_0 + c_1 * diff_z + c_2 * power_of<2>(diff_z) + s_0 *
```

Comment:
I think this is incorrect. IMHO, the "constant" part does not enter multiplied by the omega's Breit-Wigner.